lib/cc/analyzer/plugins.rb

```diff
@@ -1,27 +0,0 @@
-module CC
-  module Analyzer
-    class Plugins
```

Comment:
will we still want this class for filtering plugins that are configured via the UI or a user's .codeclimate.yml, or will that live elsewhere?